packages/front-end/components/Experiment/PValueGuardrailResults.tsx

```diff
@@ -99,7 +101,7 @@ const PValueGuardrailResults: FC<{
         users,
         name,
         hasEnoughData: hasEnoughData(
-          stats.value,
+          stats?.value ?? 0,
```

Comment:
Added this and `stats?` in the helpers in `experiments.ts` because technically a user could have an undefined `stats` object returned by line 92 above.